features/promotion/managing_promotions/accessing_coupons_managment_directly_from_promotion_edit_page_on_coupon_based_promotion.feature

```diff
@@ -0,0 +1,24 @@
+@managing_promotions
+Feature: Accessing coupons management directly from promotion edit page on coupon based promotions
+    In order to facilitate work with the management of coupons
+    As an Administrator
+    I want to access the coupons management directly from the promotion show page
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And there is a promotion "Christmas sale"
+        And it is a coupon based promotion
+        And I am logged in as an administrator
+
+    @ui
+    Scenario: Being able to manage promotion's coupons
+        When I want to modify a "Christmas sale" promotion
+        Then I should be able to manage its coupons
```

Comment:
I miss the scenario `Accessing the coupons management from the promotion page`, something like:

```
When I modify a "Christmas sale" promotion
And I want to manage this promotion's coupons
Then I should be on this promotion's coupons management page  
```

that would check does this button really work 😄 Right now we're only sure it exists.